### DISCUSSION TITLE
k3s log issue while using vagrant

### DISCUSSION THREAD

**Discussion by Milan-Talapada** [k3s-io/k3s#7201] [Q&A]
**Environmental Info:**
K3s Version: 
v1.26.3+k3s1

Node(s) CPU architecture, OS, and Version: 
Linux alice 4.4.0-210-generic #242-Ubuntu 

Cluster Configuration:
1 master 1 worker

**Describe the bug:**
stream logs failed the server could not find the requested resource ( pods/log coredns-7c444649cb-tgsrc) for kube-system/coredns-7c444649cb-tgsrc (coredns)

we require log  but it showing some error

**Comment by brandond** [comment]
You're trying to get logs for a pod that doesn't exist. Either there's a typo in the name, or the pod was deleted in between the time you listed the pods, and the time you tried to retrieve logs for it.

**Reply by Milan-Talapada** [reply]
![Screenshot from 2023-04-04 10-54-50](https://user-images.githubusercontent.com/105209339/229695429-90f0c2e2-2975-479f-8a00-4c0d62c62d5c.png)
![Screenshot from 2023-04-04 10-55-11](https://user-images.githubusercontent.com/105209339/229695433-4fbe3f07-4705-4118-bead-4468e5f51f4e.png)

**Reply by Milan-Talapada** [reply]
pod is showing i shared screenshot for y'r reference.  and also log not showing default kube system like coredns, metics server

**Reply by brandond** [reply]
How exactly did you install and start k3s? Are you using any extra CLI args or entries in the config file? What do you see in containerd.log and the k3s service log?

**Comment by Milan-Talapada** [comment]
i installed though `curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" INSTALL_K3S_EXEC="server --disable=traefik --node-external-ip=<IP>" sh - `

Here i used one master node and one worker node  when i deployment some application than  log showing  in k9s only one node either master or worker and other one showing error steam log fail 

[k3s.log](https://github.com/k3s-io/k3s/files/11158857/k3s.log)
[k3s.log](https://github.com/k3s-io/k3s/files/11158923/k3s.log)
[containerd.log](https://github.com/k3s-io/k3s/files/11158932/containerd.log)

**Reply by brandond** [reply]
How do you have the networking configured between the two nodes? I suspect that they're not able to reach each other via the default Vagrant interfaces.

**Comment by Milan-Talapada** [comment]
[vagrantfileq.docx](https://github.com/k3s-io/k3s/files/11168310/vagrantfileq.docx)

i attached vagrant file for your reference and i used  bridge network(LAN).
